src/Mealz/AccountingBundle/Tests/Controller/CostSheetControllerTest.php

```diff
@@ -4,56 +4,69 @@
 
 use App\Mealz\AccountingBundle\DataFixtures\ORM\LoadTransactions;
 use App\Mealz\AccountingBundle\Entity\Transaction;
-use App\Mealz\MealBundle\DataFixtures\ORM\LoadCategories;
-use App\Mealz\MealBundle\DataFixtures\ORM\LoadDays;
-use App\Mealz\MealBundle\DataFixtures\ORM\LoadDishes;
-use App\Mealz\MealBundle\DataFixtures\ORM\LoadDishVariations;
-use App\Mealz\MealBundle\DataFixtures\ORM\LoadMeals;
-use App\Mealz\MealBundle\DataFixtures\ORM\LoadWeeks;
+use App\Mealz\AccountingBundle\Service\Wallet;
+use App\Mealz\MealBundle\Entity\Participant;
 use App\Mealz\MealBundle\Tests\Controller\AbstractControllerTestCase;
 use App\Mealz\UserBundle\DataFixtures\ORM\LoadRoles;
 use App\Mealz\UserBundle\DataFixtures\ORM\LoadUsers;
+use DateTime;
 
 /**
  * Class CostSheetControllerTest.
  */
 class CostSheetControllerTest extends AbstractControllerTestCase
 {
+    private Wallet $wallet;
+
     protected function setUp(): void
     {
         parent::setUp();
 
         $this->clearAllTables();
         $this->loadFixtures([
-            new LoadWeeks(),
-            new LoadDays(),
-            new LoadCategories(),
-            new LoadDishes(),
-            new LoadDishVariations(),
-            new LoadMeals(),
             new LoadRoles(),
             new LoadUsers(self::$container->get('security.user_password_encoder.generic')),
             new LoadTransactions(),
         ]);
 
+        $participantRepo = $this->getDoctrine()->getRepository(Participant::class);
+        $transactionRepo = $this->getDoctrine()->getRepository(Transaction::class);
+        $this->wallet = new Wallet($participantRepo, $transactionRepo);
+
         $this->loginAs(self::USER_KITCHEN_STAFF);
     }
 
     /**
      * Check if hashCode was written in database.
```

Comment:
It would be nice to update the the test description. It's hard to figure out what the test actually testing.

Reply:
Agree. Will update the descriptions